build.gradle

```diff
@@ -211,6 +210,7 @@ subprojects {
                 // exclude generated Protobuf classes
                 exclude "**/*Proto.java"
                 exclude "**/*ProtoV3.java"
+                exclude "**/*Protos.java"
```

Comment:
What happened here? Did the name of some of the protobuf files change, or where there always generated files suffixed with `Protos.java`

Reply:
It was trying to Javadoc `DescriptorProtos.java`; perhaps that isn't the case any more after other improvements. Should confirm that it's still needed.

Reply:
`gradle javadoc` does pass in this branch without that change, indeed suggesting that it isn't needed any more.